atom/browser/browser_mac.mm

```diff
@@ -148,6 +150,28 @@
   return prevent_default;
 }
 
+v8::Local<v8::Value> Browser::GetLoginItemStatus(mate::Arguments* args) {
+  bool hidden = false;
+  mate::Dictionary dict = mate::Dictionary::CreateEmpty(args->isolate());
+  dict.Set("openAtLogin", base::mac::CheckLoginItemStatus(&hidden));
+  dict.Set("openAsHidden", hidden);
+  dict.Set("restoreState", base::mac::WasLaunchedAsLoginItemRestoreState());
+  dict.Set("openedAtLogin", base::mac::WasLaunchedAsLoginOrResumeItem());
```

Comment:
I was a little confused by the different tenses here—I wonder if calling the keys `wasOpenedAtLogin` or `didOpenAtLogin` might make it more clear?